QGCSetup.pri

```diff
@@ -40,8 +44,10 @@ WindowsBuild {
     BASEDIR_COPY_RESOURCE_LIST = $$replace(BASEDIR,"/","\\")
     QMAKE_POST_LINK += $$escape_expand(\\n) $$QMAKE_COPY_DIR \"$$BASEDIR_COPY_RESOURCE_LIST\\flightgear\" \"$$DESTDIR_COPY_RESOURCE_LIST\\flightgear\"
 } else {
-    # Make sure to keep both side of this if using the same set of directories
-    QMAKE_POST_LINK += && $$QMAKE_COPY_DIR $$BASEDIR/flightgear $$DESTDIR_COPY_RESOURCE_LIST
+    !AndroidBuild {
+        # Make sure to keep both sides of this if using the same set of directories
+        QMAKE_POST_LINK += && $$QMAKE_COPY_DIR $$BASEDIR/flightgear $$DESTDIR_COPY_RESOURCE_LIST
```

Comment:
Yeah, I noticed a few of these things too. I figured at some point we need to go through a dead code hunt. This will come when we get everything to QML. For instance, SerialSettingsDialog (.h, .cc, .ui) are only used by TerminalConsole, and even there, it doesn't do the right thing.